From 45fab34cb60bc221e6debdf9457d382b62cf4017 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 5 Aug 2015 23:36:13 +0100
Subject: [PATCH 05/16] master Windows: New feature to allow overriding
 -lmsvcrt

Added in support of the MinGW-w64 WIP feature "agile mscvrt dll" where
a process' loaded msvc runtime is used by a newly loaded DLL rather than
always using msvcrt.dll
---
 gcc/config/i386/cygming.opt | 3 +++
 gcc/config/i386/mingw32.h   | 2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/gcc/config/mingw/cygming.opt b/gcc/config/mingw/cygming.opt
index 12435d813f5..5dff2ac1551 100644
--- a/gcc/config/mingw/cygming.opt
+++ b/gcc/config/mingw/cygming.opt
@@ -22,6 +22,9 @@ mconsole
 Target RejectNegative
 Create console application.
 
+mcrtdll=
+Target RejectNegative Joined
+
 mdll
 Target RejectNegative
 Generate code for a DLL.
-- 
2.38.1

